netlogo-gui/src/main/app/AppTabManager.scala

```diff
@@ -688,6 +722,27 @@ class AppTabManager(val appTabsPanel:          Tabs,
     }
   }
 
+  // For a Menu - prints Menu Items and their Accelerators
+  def __printMenuItem(menuItem: javax.swing.JMenuItem, level: Int): Unit = {
+    if (menuItem == null) { return}
+    if (menuItem.isInstanceOf[javax.swing.JMenu]) {
+      __printMenuItems(menuItem.asInstanceOf[javax.swing.JMenu], 1);
+    } else {
+      println(menuItem.getText)
+      val accelerator = menuItem.getAccelerator
+      if (accelerator != null) {
+        println(indent(level * 2) + "Accelerator: " + accelerator);
+      }
+    }
+  }
+
+  def __printMenuItembyNameAndMenuName(menuName: String, menuItemName: String): Unit = {
+    getMenuItembyNameAndMenuName(menuName, menuItemName) match {
+      case None       => println("Menu Item '" + menuName + ":" + menuItemName + "' does not exist.")
+      case Some(menuItem) => __printMenuItem(menuItem, 1)
```

Comment:
Either the `=>` should align with each-other (preferred), or the `None` case should be with one space, but it shouldn't be in-between like this.